### ISSUE TITLE
Font colors in mail pane

### ISSUE THREAD

**Issue by paul1149** [de-vri-es/thunderbird-real-system-theme#4]
Is it possible to alter the font colors in the mail body pane? No matter which theme I've tried the links are a darkish blue, which doesn't contrast enough from the dark background, making it hard to read.

Thanks.

**Comment by de-vri-es** [comment]
Maybe! But this theme doesn't choose any fixed colors, it's supposed to let the system theme decide it. So it depends on why the color is hard to read. If it's from your system theme the fix could be there.

But maybe we need to add an override here somewhere to force the system color. Could you show on a screenshot exactly which text you mean?

**Comment by paul1149** [comment]
Thanks for getting back to me. :) 
That could explain why nothing was changing no matter the them. I'll attach a small shot of the problem - not the worst example.
Also, I checked under System Settings / Global Theme / Colors and for "Link" it says "Varies", so I was loath to change it.

![image](https://github.com/user-attachments/assets/3d554cc7-de55-4860-9633-bd59a69f040b)

**Comment by de-vri-es** [comment]
I see. I *think* this is not something that is in control of this theme. It looks like it's behaving correctly, but that your system theme has some colors that are not optimal.

It *could* also be that this theme is assigning the wrong system color to certain UI elements. But then we need to pinpoint which mixup is happening. For that, it would be good if you can compare with a native app that shows something similar.

**Comment by paul1149** [comment]
It does comport to what the system theme looks like, so I tend to think that's what's going on here. Here's a shot from System Settings / Global Theme / Colors: 

![image](https://github.com/user-attachments/assets/5655898a-e304-429d-8148-bb1e2df2a9a0)

**Comment by de-vri-es** [comment]
Ok, thanks for the clarification.

Your best bet then is to modify the system theme if you can.

Otherwise, you can also create a copy of this theme and tweak the colors you care about to something else :)

**Comment by paul1149** [comment]
Ok. Thanks much. I will look into those things. Be well.

**Comment by de-vri-es** [comment]
Good luck! You too!

**Comment by paul1149** [comment]
I wanted to update for everyone's edification. Playing with the KDE system theme did nothing. Then I looked into Thunderbird's (actually, Betterbird's) own settings. Under General / Fonts and Colors / Colors I was able to create my own scheme. I had to uncheck Use System colors, which at first created a problem because almost everything was jet black. But I found the Color settings again, reverted to system colors, made appropriate changes to TB's background, and then, unchecking Use System Colors, I was on my way to a easy-reading theme. Thanks again for your guidance!
